{{cookiecutter.repo_name}}/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,60 @@
+## For the Submitter
+Please make sure that you have followed all of the steps below, as
+it will help your PR be merged smoothly.
```

Comment:
Nit: maybe add a sentence such as:

{quote}Feel free to remove any sections that are not relevant for reviewers of your PR, including this one.{quote}